### PR TITLE
I18n: update zh-cn and zh-tw

### PR DESCRIPTION
**Pull request by Hisssy**
**Description**
Update Simplified Chinese and Traditional Chinese translation.

### CODE REVIEW

File: frontend/src/i18n/zh-cn.json [hunk 1]
```diff
@@ -90,8 +90,8 @@
     "copy": "复制",
     "copyMessage": "请选择欲复制至的目录：",
     "currentlyNavigating": "当前目录：",
-    "deleteMessageMultiple": "你确定要删除这 {count} 个文件吗?",
-    "deleteMessageSingle": "你确定要删除这个文件/文件夹吗?",
+    "deleteMessageMultiple": "你确定要删除这 {count} 个文件吗？",
+    "deleteMessageSingle": "你确定要删除这个文件/文件夹吗？",
```
Comment (o1egl) [inline frontend/src/i18n/zh-cn.json:94]: Why did you add a space at the end?

File: frontend/src/i18n/zh-cn.json [hunk 2]
```diff
@@ -112,31 +112,38 @@
...
+    "brandingHelp": "您可以通过改变实例名称，更换Logo，加入自定义样式，甚至禁用到Github的外部链接来自定义File Browser的外观和给人的感觉。\n想获得更多信息，请查看 {0} 。",
     "admin": "管理员",
     "administrator": "管理员",
-    "allowCommands": "执行命令(Linux 代码)",
+    "allowCommands": "执行命令（shell 命令）",
     "allowEdit": "编辑、重命名或删除文件/目录",
     "allowNew": "创建新文件和目录",
     "allowPublish": "发布新的帖子与页面",
-    "avoidChanges": "(留空以避免更改)",
+    "avoidChanges": "（留空以避免更改）",
```
Comment (o1egl) [inline frontend/src/i18n/zh-cn.json:141]: same

File: frontend/src/i18n/zh-cn.json [hunk 3]
```diff
@@ -209,18 +216,22 @@
...
     "it": "Italiano",
     "fr": "Français",
     "pt": "Português",
-    "ptBR": "Português (Brasil)",
+    "ptBR": "Português（Brasil）",
     "ja": "日本語",
-    "zhCN": "中文 (简体)",
-    "zhTW": "中文 (繁體)",
+    "zhCN": "中文（简体）",
+    "zhTW": "中文（繁體）",
```
Comment (o1egl) [inline frontend/src/i18n/zh-cn.json:226]: same

**Review by o1egl — CHANGES_REQUESTED** [review]
Please remove surrounding spaces

File: frontend/src/i18n/zh-cn.json (hunk 1 above)
Comment (Hisssy) [reply frontend/src/i18n/zh-cn.json:94]: Thanks for your comment.  That is not spaces, but special characters called Fullwidth Characters in CJK computing, which helps us read characters more easily when combining with Chinese characters. Furthermore, it is the common way we use in strings having Chinese. Please see [Halfwidth and fullwidth forms](https://en.wikipedia.org/wiki/Halfwidth_and_fullwidth_forms) for more information.

e.g.
"" and “”
() and （）
! and ！
. and 。

See the differences?
